Spell_Chech_Example.c

```diff
@@ -38,43 +39,67 @@ _spell_check_word(const char* str)
 
 
 /************ EFL Spell Check Logic ******************/
+
+static Efl2_Text_Attribute_Handle*
+mark_misspelled(Eo *start,Eo *end)
+{
+   efl2_text_style_underline_color_set(attribute_factory, 255, 0, 0, 255);
```

Comment:
I'd move this to the beginning of `_ui_text_spell_check_cb` and only have it set once. You'd probably need to also `clear` before in case you used the attribute factory for something else.